terraform.template.tfvars

```diff
@@ -2,12 +2,22 @@ aws_key           = "{{aws_key}}"
 aws_secret        = "{{aws_secret}}"
 digger_aws_key    = "{{digger_aws_key}}"
 digger_aws_secret = "{{digger_aws_secret}}"
-network_name               = "{{network_name}}"
+vpc_name          = "{{vpc_name}}"
+environment       = "{{environment}}"
+network_name      = "{{network_name}}"
+
+{{ 'one_nat_gateway_per_az='+one_nat_gateway_per_az | lower if one_nat_gateway_per_az is defined else '' }}
+{{ 'enable_dns_hostnames='+enable_dns_hostnames | lower if enable_dns_hostnames is defined else '' }}
+{{ 'enable_dns_support='+enable_dns_support | lower if enable_dns_support is defined else '' }}
+{{ 'enable_nat_gateway='+enable_nat_gateway | lower if enable_nat_gateway is defined else '' }}
 
 region = "{{region}}"
 
 tags = {
```

Comment:
Aren't we ignoring tags passed here to use the default tags on the provider level?

Reply:
as far as I understand provider level tags should be merged with any tags we've got in code, @Spartakovic correct me if I'm wrong